# Remove automatic call to configure at the end of autoconf which
# causes build error with trailing &&
# Author: Davide Vanzo (Vanderbilt University)
diff -ru mpb-1.6.2.orig/autogen.sh mpb-1.6.2/autogen.sh
--- mpb-1.6.2.orig/autogen.sh	2018-11-06 07:49:05.271782178 -0600
+++ mpb-1.6.2/autogen.sh	2018-11-06 07:49:21.787781954 -0600
@@ -4,5 +4,3 @@
 autoreconf --verbose --install --symlink --force
 autoreconf --verbose --install --symlink --force
 autoreconf --verbose --install --symlink --force
-
-./configure --enable-maintainer-mode "$@"
